service/controller/v24/adapter/cpf.go

```diff
@@ -0,0 +1,31 @@
+package adapter
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+)
+
+// CPF is the adapter collection for the Control Plane Finalizer management.
+type CPF struct {
```

Comment:
There is a resource coming following this scheme in https://github.com/giantswarm/aws-operator/pull/1425.